forge-gui/res/cardsfolder/upcoming/dance_with_calamity.txt

```diff
@@ -0,0 +1,11 @@
+Name:Dance with Calamity
+ManaCost:7 R
+Types:Sorcery
+A:SP$ Shuffle | SubAbility$ DBRepeat | StackDescription$ {p:You} shuffles their library, | SpellDescription$ Shuffle your library. As many times as you choose, you may exile the top card of your library. If the total mana value of the cards exiled this way is 13 or less, you may cast any number of spells from among those cards without paying their mana costs.
+SVar:DBRepeat:DB$ Repeat | RepeatSubAbility$ DBDig | RepeatOptional$ True | SubAbility$ DBPlay
```

Comment:
Seems wrong since it will always force you to exile at least one

Reply:
Hmm I guess a generic optional decider before the repeat ? or is there a more elegant way ?

Reply:
Probably just do it without Repeat

\* GenericChoice:
- Exile top
      - Sub back to *
- Try play exiled

Reply:
I think I got it working this way though the choiceprompt after the shuffle is kinda weird